test-data/local/snake_oil/jobs/snake_oil_diff.py

```diff
@@ -4,7 +4,7 @@
 
 def writeDiff(filename, vector1, vector2):
     with open(filename, "w") as f:
-        for index in range(len(vector1)):
+        for index, _ in enumerate(vector1):
             node1 = vector1[index]
```

Comment:
Interesting case since we use index for vector2 as well here, but we could do: 
```suggestion
        for index, node1 in enumerate(vector1):
```

Reply:
zip to the rescue!